### ISSUE TITLE
Pgina service issue user is not authorized to logon (using existing profile)

### ISSUE THREAD

**Issue by ryanclintoncapita** [pgina/pgina#265]
Hi we are getting the message "user is not authorized to logon (using existing profile)"
Restarting the service resolves the issue temporarily

The following message is logged in windows event viewer:

Faulting application name: LogonUI.exe, version: 6.1.7601.17514, time stamp: 0x4ce79f70
Faulting module name: pGinaCredentialProvider.DLL, version: 3.1.8.0, time stamp: 0x51ad8302
Exception code: 0xc0000005
Fault offset: 0x0000000000003859
Faulting process id: 0x3d0
Faulting application start time: 0x01d0f116e9ec808a
Faulting application path: C:\Windows\system32\LogonUI.exe
Faulting module path: C:\Windows\system32\pGinaCredentialProvider.DLL
Report Id: b6f13acd-5d1b-11e5-b64b-a41f728186da

**Comment by ryanclintoncapita** [comment]
Faulting application name: wmiprvse.exe, version: 6.1.7601.17514, time stamp: 0x4ce79d42
Faulting module name: ntdll.dll, version: 6.1.7601.18247, time stamp: 0x521eaf24
Exception code: 0xc0000374
Fault offset: 0x00000000000c4102
Faulting process id: 0x980
Faulting application start time: 0x01d0f15e5cd05c6c
Faulting application path: C:\Windows\system32\wbem\wmiprvse.exe
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report Id: 0dde896c-5d52-11e5-96c0-a41f728186da

Faulting application name: LogonUI.exe, version: 6.1.7601.17514, time stamp: 0x4ce79f70
Faulting module name: ntdll.dll, version: 6.1.7601.18247, time stamp: 0x521eaf24
Exception code: 0xc0000374
Fault offset: 0x00000000000c4102
Faulting process id: 0x1444
Faulting application start time: 0x01d0f13a8696afa9
Faulting application path: C:\Windows\system32\LogonUI.exe
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report Id: cc53060d-5d2d-11e5-b64b-a41f728186da

Faulting application name: LogonUI.exe, version: 6.1.7601.17514, time stamp: 0x4ce79f70
Faulting module name: pGinaCredentialProvider.DLL, version: 3.1.8.0, time stamp: 0x51ad8302
Exception code: 0xc0000005
Fault offset: 0x0000000000003859
Faulting process id: 0x1164
Faulting application start time: 0x01d0acce8cbb5eea
Faulting application path: C:\Windows\system32\LogonUI.exe
Faulting module path: C:\Windows\system32\pGinaCredentialProvider.DLL
Report Id: 025cafca-18ca-11e5-a871-a41f728186da

Faulting application name: LogonUI.exe, version: 6.1.7601.17514, time stamp: 0x4ce79f70
Faulting module name: DUI70.dll, version: 6.1.7600.16385, time stamp: 0x4a5bdf25
Exception code: 0xc0000005
Fault offset: 0x000000000001437f
Faulting process id: 0x3ec
Faulting application start time: 0x01d09c47144b3eb7
Faulting application path: C:\Windows\system32\LogonUI.exe
Faulting module path: C:\Windows\system32\DUI70.dll
Report Id: 0422d610-083f-11e5-af66-a41f728186da

**Comment by MutonUfoAI** [comment]
add pgina version and
please upload
%ProgramFiles%\pGina\log\pGina.Service.ServiceHost_log.txt
to pastebin

**Comment by ryanclintoncapita** [comment]
Pgina version 3.1.8.0

http://pastebin.com/RHPvxMgM

**Comment by MutonUfoAI** [comment]
2015-09-18 13:13:49,843 [3720|60|ERROR] pGina.Plugin.SimplifiedLogonPlugin: Object reference not set to an instance of an object.
   at SimplifiedLogonPlugin.PluginImplementation.SessionChange(SessionChangeDescription changeDescription, SessionProperties properties)

a SimplifiedLogonPlugin is unknown to me
contact the author and request help

**Comment by ryanclintoncapita** [comment]
Hi,
After a lot of testing it seems that I was barking up completely the wrong tree for this one and this issue is caused by Internet Explorer 11 or more specifically IE 11 Security patches.

I did the following:

Built a machine using a vanilla image that had IE 9 and very few security patches.
I tested and was not able to replicate the issue.
I then installed IE11 tested and got no issues.
I then patched IE 11 to Kb3032359 and the issue occurred.
I then tested the following IE versions and the issue occurred on them all:
Kb3021952 , Kb3065822 , Kb3074886 , Kb3075516

I have extensively tested this and the version of IE 11 where the issue does not occur is Kb2841134
